<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findStudentDialog</class>
 <widget class="QDialog" name="findStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пошук студента</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>382</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Студент</string>
   </property>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>31</y>
      <width>22</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ім'я:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>31</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="facultyLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>81</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Факультет:</string>
    </property>
   </widget>
   <widget class="QLabel" name="groupLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Група:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="facultyEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="groupEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>361</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="findButton">
       <property name="text">
        <string>Пошук</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevertButton">
       <property name="text">
        <string>Очистити</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Закрити</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
